WordPress/Classes/ViewRelated/Sharing/ShareAppTextActivityItemSource.swift

```diff
@@ -0,0 +1,36 @@
+/// A text-type UIActivityItemSource for the share app activity.
+///
+/// Provides additional subject string so the subject line is filled when sharing the app via mail.
+///
+final class ShareAppTextActivityItemSource: NSObject {
+    private let message: String
+
+    init(message: String) {
+        self.message = message
+    }
+}
+
+// MARK: - UIActivityItemSource
+
+extension ShareAppTextActivityItemSource: UIActivityItemSource {
+    func activityViewControllerPlaceholderItem(_ activityViewController: UIActivityViewController) -> Any {
+        // informs the activity controller that the activity type is text.
+        return String()
+    }
+
+    func activityViewController(_ activityViewController: UIActivityViewController, itemForActivityType activityType: UIActivity.ActivityType?) -> Any? {
+        return message
+    }
+
+    func activityViewController(_ activityViewController: UIActivityViewController, subjectForActivityType activityType: UIActivity.ActivityType?) -> String {
+        return .defaultSubjectText
+    }
+}
+
+// MARK: - Localized Strings
+
+private extension String {
+    static let defaultSubjectText = NSLocalizedString("WordPress.com Apps - Apps for any screen",
```

Comment:
@aerych, @mattmiklic are we good to go with this subject text? This text is used as the email subject line when sharing the app via Mail.app.

(Also, worth noting that Gmail.app somehow ignores this value, and takes the text content instead. 😅 )

Reply:
This is fine with me unless @aerych wants to wordsmith it. 😄

Reply:
Maybe `WordPress Apps` rather than `WordPress.com Apps`?  If we omit the .com it might sound a tad more inclusive for self-hosted/Jetpack sites.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17006/commits/b963748036929e771fd66ee58a740f63515e84ef.